e2e_deployment_files/stress_deployment.template.json

```diff
@@ -75,12 +78,15 @@
                 "value": "<LoadGen.MessageFrequency>"
               },
               "transportType": {
-                "value": "<LoadGen2.TransportType>"
+                "value": "<TransportType2>"
               }
             },
             "settings": {
               "image": "<Container_Registry>/microsoft/azureiotedge-load-gen:<Build.BuildNumber>-linux-<Architecture>",
               "createOptions": ""
+            },
+            "ServiceClientConnectionString": {
```

Comment:
why do you pass ServiceClientConnectionString for loadGen module?

Reply:
I added this when I was attempting to integrate twin tests with load-gen. I cleaned up load-gen source, but forgot to remove from deployment. Will fix.